docs/man/docker-build.1.md

```diff
@@ -37,13 +37,14 @@ daemon, not by the CLI, so the whole context must be transferred to the daemon.
 The Docker CLI reports "Sending build context to Docker daemon" when the context is sent to 
 the daemon.
 
-When a single Dockerfile is given as the URL, then no context is set.
-When a Git repository is set as the **URL**, the repository is used
-as context.
+When the URL to a tarball archive or to a single Dockerfile is given, no context is sent from
+the client to the Docker daemon. When a Git repository is set as the **URL**, the repository is
+cloned locally and then sent as the context.
 
 # OPTIONS
 **-f**, **--file**=*PATH/Dockerfile*
-   Path to the Dockerfile to use. If the path is a relative path then it must be relative to the current directory. The file must be within the build context. The default is *Dockerfile*.
+   Path to the Dockerfile to use. If the path is a relative path and you are building from a local
+   directory, then the path must be relative to that directory. If you are building from a remote URL poiting to either a tarball or a Git repository, then the path must be relative to the root of the remote context. In all cases, the file must be within the build context. The default is *Dockerfile*.
```

Comment:
@SvenDowideit  You mean the sentence "Path to the Dockerfile to use"? If so, tar archives are not accepted there, the value supplied to the `-f` option is always interpreted as pointing to a plain text Dockerfile inside the context (which can itself be a tar archive, but that does not change the rules for `-f`).